checkstyle/suppressions.xml

```diff
@@ -52,4 +52,5 @@
     <suppress checks="ClassDataAbstractionCoupling" files=".*Test\.java"/>
     <suppress checks="JavaNCSS" files=".*Test\.java"/>
     <suppress checks="ClassFanOutComplexity" files=".*Test\.java"/>
+    <suppress checks="LineLengthCheck" files="PostgreSqlDatabaseDialectTest.java"/>
```

Comment:
Can't the file just be formatted to pass the test? Now it's the new code in this PR that fails the test? This configuration removes testing of the file completely, which kind of too much.

Reply:
This is old code, we already have an issue with explanation what needs to be done to improve code of this connector. So far it is okay.

Reply:
Anyways, fixed.